read-skipfile-results

```diff
@@ -0,0 +1,433 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vim: set ts=4
+#
+# Copyright 2023-present Linaro Limited
+#
+# SPDX-License-Identifier: MIT
+
+from argparse import ArgumentParser
+from copy import deepcopy
+from datetime import datetime
+from json import load
+from logging import DEBUG, INFO, basicConfig, getLogger
+from os import chdir, environ, getenv, path, popen, rename
+from pathlib import Path
+from re import findall, sub
+
+import pandas as pd
```

Comment:
the requirements.txt file should be updated with pandas in this commit.

Reply:
I have updated the requirements.txt to reflect the requirements of the scripts